docs/content/guides/dagster/model_selection.mdx

```diff
@@ -0,0 +1,231 @@
+
+     sepal length (cm)  sepal width (cm)  petal length (cm)  petal width (cm)  target
+                   5.1               3.5                1.4               0.2       0
+                   4.9               3.0                1.4               0.2       0
+                   4.7               3.2                1.3               0.2       0
+                   4.6               3.1                1.5               0.2       0
+                   5.0               3.6                1.4               0.2       0
+
+The classification task is to predict the `target` using the physical measurements. Because this is a [supervised learning problem](https://machinelearningmastery.com/supervised-and-unsupervised-machine-learning-algorithms/), we have a clear model performance metric: the fraction of records correctly classified by the model. We'll call this number (ranging from 0-1) the _classification accuracy_.
+
+The model selection problem is then to select a model architecture that maximizes the classification accuracy. Recall that a "model architecture" here means "a learning algorithm together with bindings for its hyperparameters". Our candidate architectures will be based on two classification algorithms: [Logistic Regression](https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression) and [Random Forest](https://scikit-learn.org/stable/modules/ensemble.html#forest). For each algorithm, we'll generate candidate model architectures by iterating over a fixed set of hyperparameter bindings (i.e we'll perform a [Grid Search](https://machinelearningmastery.com/hyperparameter-optimization-with-random-search-and-grid-search/)). For each model architecture, we'll apply 5-fold [cross-validation](https://scikit-learn.org/stable/modules/cross_validation.html) to obtain a classification accuracy score. Finally, we'll select the architecture with the highest score, use it to train a model on the entire training set, and calculate it's final score against the test set.
+
+## Dagster solution
+
+We're going to implement our solution using a combination of Dagster and popular machine learning library [Scikit Learn](https://scikit-learn.org/stable/index.html) (`sklearn`). We'll implement the orchestration aspects of our solution in Dagster, and outsource the fitting of individual models to `sklearn`. (Note: `sklearn` has its own powerful and concise API for model selection, as well as a limited pipeline API. Both are useful, but Dagster gives us much greater control over and observability of our computation graph.)
+
+Let's start with an overview of the finished Dagster job we will build. Here is what the completed job looks like in [Dagit](/concepts/dagit/dagit):
+
+<Image
```

Comment:
Including this visual representation of the graph is a nice touch - I think will be very helpful.